src/renderers/webgl/WebGLUtils.js

```diff
@@ -51,6 +51,7 @@ function WebGLUtils( gl, extensions ) {
 		if ( p === LuminanceAlphaFormat ) return gl.LUMINANCE_ALPHA;
 		if ( p === DepthFormat ) return gl.DEPTH_COMPONENT;
 		if ( p === DepthStencilFormat ) return gl.DEPTH_STENCIL;
+		if ( p === RedFormat ) return gl.RED;
```

Comment:
`gl.RED` is available only in WebGL2 so I think we need to check if `gl` is `WebGL2RenderingContext`.

    if ( capabilities.isWebGL2 && p === RedFormat ) return gl.RED;